lib/App/Config/Chronicle.pm

```diff
@@ -399,13 +362,34 @@ sub loaded_revision {
     return $self->data_set->{version};
 }
 
+sub _application_settings {
+   my $self = shift;
+
+   my $redis = BOM::Config::Redis::redis_replicated_read();
```

Comment:
This is a public module. Shouldn't have dependency on `BOM::*` repositories. Why not use `$self->chronicle_reader`?